contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -590,10 +692,8 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
                     } else if (start + range.minted - 1 >= _tokenId) {
                         // Is token in target's minted range?
 
-                        // If token does not exist, is in the range, but exchange also exists, it is not committable anymore
-                        // This happens when preminted voucher was already redeemed/revoked/canceled and therefore burned
-                        (bool exists, ) = getBosonExchange(_tokenId);
-                        if (!exists) {
+                        // It is committable if it has not been burned
+                        if (_tokenId > range.lastBurnedTokenId) {
```

Comment:
is this referring to preMint'd burns specifically ? As normal burns are not necessarily incrementing in terms of the tokenId of the burned tokens.

Reply:
great, am happy that it is in the tests.